app/components/KanIkkeSendes/KanIkkeSendes.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from "./KanIkkeSendes.module.css";
+import { Alert } from "@navikt/ds-react";
+
+interface IProps {
+  kanSendes: boolean;
+  children: React.ReactNode;
+}
+
+export function KanIkkeSendes(props: IProps): JSX.Element | undefined {
```

Comment:
Hadde renamet denne til noe sånn: ForTidligInnsending eller noe som har Alert suffix

Reply:
Den brukes ikke nødvendigvis om det er for tidlig, det kan også være at meldekortet allerede er sendt inn. Og så ville jeg ikke bruke `Alert` i navnet, i tilfelle komponenten ikke alltid er en alert